ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -39,9 +40,33 @@ public override ReactTextShadowNode CreateShadowNodeInstance()
         /// <param name="extraData">The aggregated virtual node changes.</param>
         public override void UpdateExtraData(TextBlock root, object extraData)
         {
-            var inline = (Inline)extraData;
-            root.Inlines.Clear();
-            root.Inlines.Add(inline);
+            var textUpdate = default(Tuple<Inline, TextAlignment?, int?, double?>);
```

Comment:
See comment in shadow node re: using an Action<TextBlock>, perhaps it would be more maintainable?